### PR TITLE
Create basic MVC framework for Orders + Set Up "Your Account" Section Upon Login

### PR DESCRIPTION
**Pull request by itsaliu90**
- Reworked "Members Only" boilerplate from Joe to "Your Account" to be more appropriate for B.S. Movies - specifically removing the kitten :smirk_cat: generator and some renaming
- Created Order model in mongoose
- Link from "Your Account" to `/order` route and view
- Framework code for Order controllers and factories

_Also,_
- Fixed bug when there were multiple items in cart and you deleted one, as Randy pointed out in https://github.com/carolynyoo/stack_store/issues/74
- Added and applied a `app.filter` for cents-to-dollars conversion on Cart view :moneybag: 

As always, a video :fallen_leaf: :
order

### CODE REVIEW

File: browser/js/members-only/members-only.js [hunk 1]
```diff
@@ -1,32 +0,0 @@
-app.config(function ($stateProvider) {
```
Comment (carolynyoo) [inline browser/js/members-only/members-only.js:1]: do we still need this file at all?

File: browser/js/cart/cart.js [hunk 2]
```diff
@@ -45,4 +45,12 @@ app.factory('cartFactory', function ($http) {
 			});
 		}
 	};
-});
\ No newline at end of file
+});
+
+// Filter for cents -> dollars
+
+app.filter("centsToDollars", function() {
```
Comment (carolynyoo) [inline browser/js/cart/cart.js:52]: lets also get a TOTAL up in here :smiley_cat:

File: browser/js/members-only/members-only.js (hunk 1 above)
Comment (itsaliu90) [inline browser/js/members-only/members-only.js:1]: Nope, the file is actually getting deleted!! I think that's just the way Github displays it. :sake:

File: browser/js/cart/cart.js (hunk 2 above)
Comment (itsaliu90) [inline browser/js/cart/cart.js:52]: Yes absolutely, in the next story :). A question to decide though - are we going to let people purchase multiple of each movie? We'll have to update our models a bit to accommodate that then...

Comment (darksim735) [inline browser/js/cart/cart.js:52]: :hamburger:

**Comment by carolynyoo** [comment]
:+1:

**Pull request merged**